cli/util/osfs.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+	"io/fs"
+	"os"
+)
+
+type osFS struct{}
+
+var osfs osFS
+
+func GetOsFS() fs.FS {
+	return osfs
+}
+
+func (fs osFS) Open(name string) (fs.File, error) {
+	return os.Open(name)
+}
+
+func (fs osFS) ReadFile(name string) ([]byte, error) {
+	return os.ReadFile(name)
+}
```

Comment:
To be honest, I didn't get the idea. Why do we need this stuff?

Reply:
There is interface fs.FS (as well as extended ones like fs.ReadFileFS), but there is no default implementation. Here it is. It should be used wherever you need to be able to substitute non-default FS. The most obvious scenario is using mock FS for testing (like this PR do). Say we have some function that expect path as an argument and we need to test it. To be able to use mock FS in such a case we need to add `fsys FS` argument to this function and replace `os.Open/os.ReadFile` within function to `fsys.Open/fsys.ReadFile`.

Reply:
Application code call the mentioned function like
```go
someFunction(utils.GetOsFS(), somePath)
```
while tests are able to substitute mockFS
```go
someFunction(someMockFS, somePath)
```

Reply:
Probably you could to use library calls instead of a custom implementation:

https://pkg.go.dev/os#DirFS
https://pkg.go.dev/os#Root.FS